src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,146 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
+    static class Node<T> {
```

Comment:
Add access modifier. Also, locate this static inner class under all methods.

Reply:
not fixed

Reply:
I found out, what does it mean? "Add access modifier. Also, locate this static inner class under all methods."